installer/bundle/bundle_skel_Linux.sh

```diff
@@ -513,10 +521,23 @@ then
 
     if [ "$installMode" = "P" ]
     then
-        echo "Purging all files in cross-platform agent ..."
-        rm -rf /etc/opt/microsoft/*-cimprov /etc/opt/microsoft/scx /opt/microsoft/*-cimprov /opt/microsoft/scx /var/opt/microsoft/*-cimprov /var/opt/microsoft/scx
-        rmdir /etc/opt/microsoft /opt/microsoft /var/opt/microsoft 1>/dev/null 2>/dev/null
-
+        check_if_pkg_is_installed apache-cimprov
```

Comment:
Adding onto Jeff's comment about keeping these provider references generic, perhaps these two -cimprov checks can be eliminated and combined into the scx check like the OMS code does [here](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/uninstall.sh#L109)